modules/katamari/src/main/ts/ephox/katamari/api/Type.ts

```diff
@@ -1,10 +1,26 @@
+const getPrototypeOf = Object.getPrototypeOf;
+
+const hasProto = (v: Object, predicate: (v: Object) => boolean, name: string): boolean => {
+  if (predicate(v)) {
+    return true;
+  } else {
+    // String-based fallback time (only needed for IE)
```

Comment:
This comment still isn't very clear, as now it sounds like the `constructor` check is also only needed for IE 11, but it's needed for modern browsers. It's only the `toString` check that's needed for IE 11.

Reply:
Good catch, I've moved the "IE only" comment onto the code that's actually IE only